demo-java/local.defaults.properties

```diff
@@ -0,0 +1 @@
+PLACES_API_KEY="YOUR_API_KEY"
```

Comment:
Consider adding a line MAPS_API_KEY="YOUR_API_KEY" in case the app uses Maps SDK as well and has a different API key for their Maps SDK usage. (I had to do this when adding the Address Form with confirmation map sample.